Makefile

```diff
@@ -96,9 +96,10 @@ test_unit: clean static_external
 	@echo -e "\nCoverage HTML report at file://`pwd`/build/coverage/index.html\n"
 	@coverage report --fail-under 94 || (echo "\nERROR: Coverage is below 94%\n" && exit 2)
 
-# Check whether migrations need to be generated
+# Check whether migrations need to be generated, creating "opencraft" database first if it doesn't exist
 test_migrations_missing: clean
-	! honcho -e .env.test run ./manage.py makemigrations --dry-run --exit
+	createdb --encoding utf-8 --template template0 opencraft 2> /dev/null || true
```

Comment:
Can this be put in its own `create_db` rule, and the rule added as a dependency for test_migrations_missing? Or, better, rather than always get `test_migrations_missing` to execute it, add `make create_db` just for circleci in https://github.com/open-craft/opencraft/blob/85726195592454e3800565988ca61a22710b2a0f/circle.yml#L15  and replace https://github.com/open-craft/opencraft/blob/master/bin/bootstrap#L52-L53 to factorize.